src/helpers/searchCensusHelper.ts

```diff
@@ -1,19 +1,19 @@
 import topics from "../data/content";
 
 export const searchCensus = (q: string) => {
-  let s = q.toLowerCase();
+  const s = q.toLowerCase();
```

Comment:
Correct, but that's just an `--autofix` coming from `eslint:recommended` that we have in place.

These turn into consts because they are never reassigned. If we don't want that rule, then the config should be updated accordingly.

Reply:
Just noticed this is actually happening everywhere:

```
/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/app.d.ts
   6:13  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
   8:13  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
  10:13  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
  12:13  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/data/fetchVizData.ts
  16:7  error  'vizData' is never reassigned. Use 'const' instead     prefer-const
  28:7  error  'bboxParam' is never reassigned. Use 'const' instead   prefer-const
  29:7  error  'url' is never reassigned. Use 'const' instead         prefer-const
  30:7  error  'response' is never reassigned. Use 'const' instead    prefer-const
  31:7  error  'csv' is never reassigned. Use 'const' instead         prefer-const
  38:7  error  'url' is never reassigned. Use 'const' instead         prefer-const
  41:7  error  'response' is never reassigned. Use 'const' instead    prefer-const
  42:7  error  'parsed' is never reassigned. Use 'const' instead      prefer-const
  49:7  error  'geoCode' is never reassigned. Use 'const' instead     prefer-const
  50:7  error  'total' is never reassigned. Use 'const' instead       prefer-const
  51:7  error  'count' is never reassigned. Use 'const' instead       prefer-const
  52:7  error  'percentage' is never reassigned. Use 'const' instead  prefer-const

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/helpers/categoryHelper.ts
   9:7  error    'topic' is never reassigned. Use 'const' instead           prefer-const
  10:7  error    'variable' is never reassigned. Use 'const' instead        prefer-const
  11:7  warning  'classification' is assigned a value but never used        @typescript-eslint/no-unused-vars
  11:7  error    'classification' is never reassigned. Use 'const' instead  prefer-const
  16:7  error    'category' is never reassigned. Use 'const' instead        prefer-const
  26:7  error    'allVariables' is never reassigned. Use 'const' instead    prefer-const
  27:7  error    'allCategories' is never reassigned. Use 'const' instead   prefer-const
  30:7  error    'match' is never reassigned. Use 'const' instead           prefer-const

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/helpers/queryParamsHelper.ts
  5:7  error  's' is never reassigned. Use 'const' instead  prefer-const

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/map/initMap.ts
  14:7   error    'map' is never reassigned. Use 'const' instead      prefer-const
  45:9   error    'geoCode' is never reassigned. Use 'const' instead  prefer-const
  53:7   error    'b' is never reassigned. Use 'const' instead        prefer-const
  54:7   error    'bbox' is never reassigned. Use 'const' instead     prefer-const
  55:7   error    'zoom' is never reassigned. Use 'const' instead     prefer-const
  63:35  warning  'zoom' is defined but never used                    @typescript-eslint/no-unused-vars

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/map/renderMapViz.ts
   7:3   error  Do not use "@ts-ignore" because it alters compilation errors  @typescript-eslint/ban-ts-comment
   8:7   error  'features' is never reassigned. Use 'const' instead           prefer-const
  11:9   error  'dataForFeature' is never reassigned. Use 'const' instead     prefer-const
  23:12  error  'b' is never reassigned. Use 'const' instead                  prefer-const

/Users/designorant/Projects/ONSdigital/Code/dp-census-atlas/src/util/numberUtil.ts
   3:1  error  Unexpected var, use let or const instead  no-var
   7:1  error  Unexpected var, use let or const instead  no-var
  11:3  error  Unexpected var, use let or const instead  no-var

✖ 38 problems (36 errors, 2 warnings)
  31 errors and 0 warnings potentially fixable with the `--fix` option.
```

Gonna revert the change here, but let's tackle linting rules separately.

Reply:
Reverted and rebased. Back to you @petem-methods.